composer.lock

```diff
@@ -62,67 +63,7 @@
                     "type": "community_bridge"
                 }
             ],
-            "time": "2020-11-19T20:18:59+00:00"
-        },
-        {
-            "name": "laminas/laminas-zendframework-bridge",
```

Comment:
This moved bellow to `packages-dev` list

Reply:
Can we drop it instead? To do that, it should suffice to bump to latest laminas-stdlib

Reply:
Already bump `laminas-stdlib` to `3.5.0` and that's why it was moved from `packages`.
But there are other packages in dev that requires it.